mypy.ini

```diff
@@ -159,26 +159,242 @@ warn_return_any = False
 [mypy-twisted.protocols.test.test_basic]
 allow_incomplete_defs = True
 
-[mypy-twisted.python.*]
```

Comment:
This replaces the `twisted.python.*` section with sections for each file, and enables as many checks as seem sufficiently straightforward.